app/views/foreman/unattended/snippets/_subscription_manager_registration.erb

```diff
@@ -11,8 +11,13 @@
   subscription-manager register --org="<%= @host.rhsm_organization_label %>" --name="<%= @host.name %>" --activationkey="<%= @host.params['kt_activation_keys'] %>"
 
   <% unless @host.operatingsystem.atomic? %>
-    echo "Installing Katello Agent"
-    yum -t -y -e 0 install katello-agent
-    chkconfig goferd on
+    <% if host_param('kt-host-tools') %>
```

Comment:
what is `host_param`? I can't find it anywhere in Katello.

Reply:
Comes from Foreman https://github.com/theforeman/foreman/blob/develop/lib/foreman/renderer.rb#L44-L47

https://groups.google.com/forum/#!topic/foreman-dev/P364HozAYZA was a discussion about it.

Reply:
Does this create this param or does it already exist in the code base? I ask cause I tend not to be a fan of shortening things. You can argue my next question should be for another discussion but since you are introducing this change. This essentially makes `katello-agent` the default, however, with the changes we've made, I'd argue that this should be flipped. The default should be goferless mode with just `katello-host-tools` and the agent mode should be an explicit opt-in now.

Reply:
This was to match the Activation Key one. https://github.com/Katello/katello/blob/master/app/views/foreman/unattended/snippets/_subscription_manager_registration.erb#L11

should i change it to `kt_host_tools` or do we want `katello_host_tools` ? I might actually change it to `(kt|katello)_agent` and make katello-host-tools default.